### ISSUE TITLE
Slider ARIA properties

### ISSUE THREAD

**Issue by aitchiss** [omgovich/react-colorful#171]
Really love this library, especially all the accessibility considerations built in! One thing I've noticed in implementing it though is an error from accessibility validators for a missing `aria-valuenow` property on both the Color and Hue sliders.

I had an explore of the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-valuetext) for the two properties `aria-valuetext` (currently used on these sliders) and `aria-valuenow`:

>The aria-valuetext attribute is used with the aria-valuenow attribute, not instead of it, unless that value is not known.
>
>aria-valuetext is only needed when the numeric value of aria-valuenow is not meaningful

The [WAI-ARIA docs for sliders](https://www.w3.org/TR/wai-aria-practices/#slider) also confirm `aria-valuenow` is required:

> Each slider element has the aria-valuenow property set to a decimal value representing the current value of the slider.

It seems that although `aria-valuetext` is what actually forms the description announced to screen reader users, the `aria-valuenow` is still required.

## Proposed change

In the case of the Hue component, the `valuetext` is already number, so I'm hoping it would be simple enough to change `aria-valuetext` to `aria-valuenow`, since the additional text description isn't needed.

It's less clear to me whether there's a corresponding numeric value that could be used for the Color slider - what do you think?
